earth_enterprise/src/fusion/portableglobe/gerewritedbroot.cpp

```diff
@@ -300,6 +324,7 @@ int main(int argc, char *argv[]) {
   std::string progname = argv[0];
   bool help = false;
   bool use_ssl_for_kml = false;
+  bool preserve_kml_filenames;
```

Comment:
This needs to be initialized to false - otherwise there's no way to know the value if it isn't specified on the command line.